stl/inc/atomic

```diff
@@ -1924,6 +2076,15 @@ using atomic_ptrdiff_t = atomic<ptrdiff_t>;
 using atomic_intmax_t  = atomic<intmax_t>;
 using atomic_uintmax_t = atomic<uintmax_t>;
 
+// 1, 2, 4 or 8 - use largest
+using atomic_int_fast_wait_t  = atomic_int64_t;

```

Comment:
I don't see these in the WP, where did they come from?

Reply:
Good catch, they participate in earlier edit [p1135r2](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1135r2.html). Now they should be removed.

Reply:
addressed